sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -108,6 +108,43 @@ def test_rfe():
     assert_array_almost_equal(X_r, X_r_sparse.toarray())
 
 
+def test_rfe_sample_weights():
+    iris = load_iris()
+    X = iris.data
+    y = iris.target
+
+    clf = SVC(kernel="linear")
+    rfe = RFE(estimator=clf, n_features_to_select=1)
+
+    sample_weight_test = 2
+    class_test = 2
+
+    # Case 1 - original dataset
+    rfe.fit(X, y)
+    ranking_original = rfe.ranking_.copy()
```

Comment:
I would prefer to clone and have an estimator for each.

```suggestion
    rfe_without_sample_weight = clone(rfe).fit(X, y)
```

You can import `clone` from `sklearn.base`